pkg/kubelet/apis/config/validation/validation_windows.go

```diff
@@ -46,5 +46,9 @@ func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration) erro
 		klog.Warningf(message, "EnforceNodeAllocatable", "--enforce-node-allocatable", kc.EnforceNodeAllocatable)
 	}
 
+	if kc.UserNamespaces != nil && kc.UserNamespaces.IDsPerPod != nil {
+		return fmt.Errorf("invalid configuration: userNamespaces.idsPerPod is not supported on Windows")
```

Comment:
if the entire userNamespaces feature is unsupported on windows, prevent use of the whole config stanza?

```suggestion
	if kc.UserNamespaces != nil {
		return fmt.Errorf("invalid configuration: userNamespaces is not supported on Windows")
```

Reply:
done